make all fails

#### What happened:
I have forked the repo and try to run make and it failed.


#### What you expected to happen:
make to succeed 

#### How to reproduce it (as minimally and precisely as possible):
```
make all
mkdir -p build
go build -ldflags '-s -w -linkmode external -extldflags "-static" -X sigs.k8s.io/security-profiles-operator/internal/pkg/version.buildDate='2021-01-12T16:15:06Z' -X sigs.k8s.io/security-profiles-operator/internal/pkg/version.gitCommit=c7f337312e70b657af8e8c51b9310c47ea74f2a7 -X sigs.k8s.io/security-profiles-operator/internal/pkg/version.gitTreeState=clean -X sigs.k8s.io/security-profiles-operator/internal/pkg/version.version=0.3.0-dev' -tags 'netgo osusergo' -o build/security-profiles-operator ./cmd/security-profiles-operator
# github.com/containers/common/pkg/seccomp
../../../../pkg/mod/github.com/containers/common@v0.29.0/pkg/seccomp/supported.go:38:12: undefined: unix.Prctl
../../../../pkg/mod/github.com/containers/common@v0.29.0/pkg/seccomp/supported.go:38:23: undefined: unix.PR_GET_SECCOMP
../../../../pkg/mod/github.com/containers/common@v0.29.0/pkg/seccomp/supported.go:40:13: undefined: unix.Prctl
../../../../pkg/mod/github.com/containers/common@v0.29.0/pkg/seccomp/supported.go:40:24: undefined: unix.PR_SET_SECCOMP
../../../../pkg/mod/github.com/containers/common@v0.29.0/pkg/seccomp/supported.go:40:45: undefined: unix.SECCOMP_MODE_FILTER
make: *** [build/security-profiles-operator] Error 2
```

#### Anything else we need to know?:
A better contributing guide would be helpful. I will do a PR to bootstrap when I can get past this.

#### Environment:

- Cloud provider or hardware configuration:
- OS (e.g: `cat /etc/os-release`): osx
- Kernel (e.g. `uname -a`):
- Others:

--- a/.github/dependabot.yml
+++ b/.github/dependabot.yml
@@ -1,0 +1,7 @@
+version: 2
+updates:
+- package-ecosystem: gomod
+  directory: "/"
+  schedule:
+    interval: daily
+  open-pull-requests-limit: 10

